cloudx/handler.go

```diff
@@ -231,6 +231,30 @@ func (h *SnakeCharmer) EnsureContext() (*AuthContext, error) {
 	}
 
 	if len(c.SessionToken) > 0 {
+		client, err := newKratosClient()
+		if err != nil {
+			return nil, err
+		}
+		_, resp, err := client.V0alpha2Api.ToSession(h.ctx).XSessionToken(c.SessionToken).Execute()
+		if resp.StatusCode == http.StatusUnauthorized || err != nil {
+			if h.isQuiet {
+				return nil, errors.New("can not sign in or sign up when flag --quiet is set")
+			}
+			ok, err := cmdx.AskScannerForConfirmation(fmt.Sprintf("Your CLI session has expired. Do you wish to log in again as \"%s\"?", c.IdentityTraits.Email), h.stdin, h.verboseErrWriter)
+			if err != nil {
+				return nil, err
+			}
+			if ok {
```

Comment:
What happens if `ok` is `false`? It looks like we now just ignore the error. Please fix this and add a test for this behavior!

Reply:
I handled this case